### ISSUE TITLE
imjournal: strange error message "sd_journal_next() failed: 'Success'"

### ISSUE THREAD

**Issue by rgerhards** [rsyslog/rsyslog#1867]
This then, at least in some cases, results in disabling the input. Sample from actual case:

```
1342.841753475:imjournal.c    : errmsg.c: Called LogMsg, msg: imjournal: ignoring invalid state file
1342.841775335:imjournal.c    : iminternal.c: signaling new internal message via SIGTTOU: 'imjournal: ignoring invalid state file [v8.30.0]'
1342.841848727:main thread    : janitor.c: janitorRun() called
1342.842293712:imjournal.c    : errmsg.c: Called LogMsg, msg: sd_journal_next() failed: 'Success'
1342.842315676:imjournal.c    : iminternal.c: signaling new internal message via SIGTTOU: 'sd_journal_next() failed: 'Success' [v8.30.0]'
1342.842344945:main thread    : janitor.c: janitorRun() called
1342.842740928:imjournal.c    : ../threads.c: thrdStarter: usrThrdMain imjournal - 0x7f8216196700 returned with iRet -3000, exiting now.
```

**Comment by Thor77** [comment]
Is this fixed with #1872 ?
Can you create a release based on that?
imjournal is basically unusable without these fixes.

**Comment by rgerhards** [comment]
I just got confirmation that it is working. It will be part of next tuesday's 8.31.0 release.
